packages/web-service/src/pages/habitat/a24/work-end/__tests__/habitat-work-end.spec.js

```diff
@@ -142,13 +147,14 @@ describe('The habitat work end page', () => {
         const payload = {
           'habitat-work-end-day': '1',
           'habitat-work-end-month': '12',
-          'habitat-work-end-year': (new Date().getFullYear()).toString()
+          'habitat-work-end-year': (new Date().getFullYear() + 1).toString()
         }
         const { validator } = await import('../habitat-work-end.js')
         expect(await validator(payload))
+        throw new NoErrorThrownError('Expected error not thrown')
       } catch (e) {
         expect(e.message).toBe('ValidationError')
-        expect(e.details[0].message).toBeNull()
+        expect(e.details[0].type).toBe('outsideLicence')
```

Comment:
why has has this changed from the message to be null ? Was this just not covering the functionality properly.

Reply:
I dunno. I could leave it in as an extra check